src/node.rs

```diff
@@ -244,12 +244,20 @@ impl<S: StorageExt> SawtoothRaftNode<S> {
 
         let is_leader = self.raw_node.raft.state == raft::StateRole::Leader;
         if is_leader {
-            // We just became the leader, so we need to start building a block
+            // We just became the leader
             if self.leader_state.is_none() {
-                debug!("Leader({:?}) became leader, intializing block", self.peer_id);
+                // Check if we stepped up as leader from follower state
```

Comment:
I think you mean "committing state"

Reply:
I meant "Check if we stepped up as leader from follower while in committing state".  I can udpate the comment if its fine ?

Reply:
Yep, that's fine

Reply:
Updated commit